quickstarts/lighttpd/config.yml

```diff
@@ -0,0 +1,41 @@
+
+slug: lighttpd
+description: |
+  ## Comprehensive monitoring for your lighttpd server
+
+  Lighttpd is an open-source web server designed for high-performance situations while staying standards-compliant, secure, and adaptable.
+
+  ### Why monitor your lighttpd server?
+
+  Lighttpd monitoring provides important information to help you construct a complete picture of your web server's performance. This performance data includes uptime, network in bytes and packets, number of connections, and more.
+
+  ### What’s included?
+
+  When you monitor your lighttpd server with New Relic, you get immediate full-stack observability, including:
+  - Be alerted on things like average number of connections, average number of requests and high memory usage.
```

Comment:
Be alerted on things like -> Alerts on

Reply:
@seemantk , Changes have been completed. Kindly review and let me know if something incorrect.
Thank you..